src/app/crash/log.h

```diff
@@ -10,7 +10,10 @@
 
 #include "base/debug.h"
 
-// TODO should we log some events in release?
```

Comment:
@dacap I changed this to work only in dev builds because it was pretty helpful to debug session issues (which we'll probably be doing for the performance problems it currently has), but I can undo it if it's not desired.

Reply:
I'd prefer to keep the `RECO_TRACE()` as it was, so we enable it only in case that we need to debug this (instead tracing all this info on Debug builds).